[Wrangler] Improve documentation for a few flags in dev/publish (#6177)
cloudflare/cloudflare-docs: content/workers/wrangler/commands.md

- `--inspector-port` {{<type>}}number{{</type>}}
  - Port for devtools to connect to.
- `--routes`, `--route` {{<type>}}string[]{{</type>}}

>   - Routes to upload. Refer to [types of routes](/workers/wrangler/configuration/#types-of-routes).

Author: ```suggestion
  - Routes to upload. 
  - For example: `--route example.com/*`
```

  - For example, `--define GIT_HASH:$(git rev-parse HEAD)` will replace all uses of `GIT_HASH` with the actual value at build time.
  - This flag is an alternative to defining [`define`](/workers/wrangler/configuration/#non-inheritable-keys) in your `wrangler.toml`. If defined in both places, this flag's values will be used.
- `--triggers`, `--schedule`, `--schedules` {{<type>}}string[]{{</type>}}
  - Cron schedules to attach to the published Worker. Refer to [cron trigger examples](/workers/platform/cron-triggers/#examples).
- `--routes`, `--route` {{<type>}}string[]{{</type>}}

>   - Routes where this Worker will be published. Refer to [types of routes](/workers/wrangler/configuration/#types-of-routes).

Author: ```suggestion
  - Routes where this Worker will be published. 
  - For example: `--route example.com/*`
```

  - For example, `--define GIT_HASH:$(git rev-parse HEAD)` will replace all uses of `GIT_HASH` with the actual value at build time.
  - This flag is an alternative to defining [`define`](/workers/wrangler/configuration/#non-inheritable-keys) in your `wrangler.toml`. If defined in both places, this flag's values will be used.
- `--triggers`, `--schedule`, `--schedules` {{<type>}}string[]{{</type>}}
  - Cron schedules to attach to the published Worker. Refer to [cron trigger examples](/workers/platform/cron-triggers/#examples).
- `--routes`, `--route` {{<type>}}string[]{{</type>}}
  - Routes where this Worker will be published. 

>   - For example: `--route example.com/*`

Author: ```suggestion
  - For example: `--route example.com/*`.
```
